### DISCUSSION TITLE
Uni logo on title page

### DISCUSSION THREAD

**Discussion by awellis** [fmarotta/kaobook#93] [Q&A]
Hi, 

has anyone got any idea how I can include my uni's logo in the top right corner of the title page? If I use `\includegraphics` in the `\titlehead`, I get the logo on the first two pages.

Andrew

**Comment by fmarotta** [comment]
Hi, you can try with something like
```
\titlehead{\hfill\includegraphics[height=3cm]{logo.jpg}}
```
as I suspect you already did. You get the logo in the first two pages only because of the line
```
\uppertitleback{\@titlehead}
```
in the copyright page (around line 100 of the file examples/documentation/main.tex). You can remove that line altogether or change `\@titlehead`, which in your case will contain the university logo, with something else.
Hope this helps!

**Comment by awellis** [comment]
Thanks, removing `\uppertitleback{\@titlehead}` was what I was looking for. Thanks for the kaobook class, this has saved me so much work 👏🏼

**Reply by johannesmols** [reply]
Hi, could you share how you positioned the logo in the top-right corner? When I use `\includegraphics` in the `titlehead` it is added below the text in the top-left corner, and I can't seem to move it around with other commands, and also can't put it inside a `figure` environment.

**Comment by awellis** [comment]
I used this
```latex
\titlehead{\includegraphics[width=4cm, right]{unilogo}}
```
and removed `\uppertitleback{\@titlehead}` as described above. As far as I can remember, that's all I had to do.


```latex
\documentclass[
	fontsize=10pt,
	twoside=false, 
	numbers=noenddot,
	bibstyle=authoryear,
	citestyle=authoryear
]{kaobook}

% Set the language
\usepackage[english]{babel} % Load characters and hyphenation
\usepackage[english=british]{csquotes} % English quotes

% Load the bibliography package
\usepackage[backend=biber,style=apa,autocite=inline]{styles/kaobiblio}
\addbibresource{bibliography.bib}

% Load mathematical packages for theorems and related environments. NOTE: choose only one between 'mdftheorems' and 'plaintheorems'.
\usepackage{styles/mdftheorems}
%\usepackage{styles/plaintheorems}


\usepackage{styles/packages}

\usepackage[export]{adjustbox}

\graphicspath{{examples/documentation/images/}{images/}} % Paths in which to look for images

%------------------------------------------------------------------------------

\begin{document}

%------------------------------------------------------------------------------
%	BOOK INFORMATION
%------------------------------------------------------------------------------

\titlehead{\includegraphics[width=4cm, right]{unilogo}}
% \subject{Use this document as a template}

\title[]{Thesis Title}
```

**Reply by johannesmols** [reply]
Thank you for the quick response! When I add the `right` keyword it gives me an error saying `Package xkeyval Error: "right" undefined in families Gin`. Apparently the `adjustbox` package has to be imported first, but it gives me the same error.

But I was now able to add the logo inside the subject field, which centers it above the title. That works as well :)

**Reply by awellis** [reply]
I think you need to use `\usepackage[export]{adjustbox}`
